app/views/themes/v2.1/contributor/registration_required.html.erb

```diff
@@ -8,10 +8,9 @@
 	
 	<h2><%= raw t('common.links.help') %></h2>
 	
-		<p><%= raw t('views.contributor.help.p1', :about => link_to( t('common.menus.main.about'), 'http://europeana.eu/portal/about', :target => '_blank') ) %></p>
+		<p><%= raw t('views.contributor.help.p1', :about => link_to( t('common.menus.main.about'), 'http://europeana.eu/portal/collections/world-war-I', :target => '_parent') ) %></p>
```

Comment:
sure, I've updated this, there are a lot of other hardcoded references to the portal app at 'europeana.eu/...' but I don't think it necessarily makes sense to update them all.